content/client-lib-development-guide/features.textile

```diff
@@ -1242,6 +1255,30 @@ h4. TokenRequest
 * @(TE4)@ @ttl@ attribute represents time to live (expiry) of this token in milliseconds
 * @(TE6)@ @TokenRequest::fromJson@ static factory method that accepts either a @JsonObject@ or a string (which should be parsed as a JSON string), and returns a new @TokenRequest@. Statically typed languages (that expect the @authCallback@ to result in a typed @TokenRequest@ object, rather than, say, a hashmap) must implement this; others may at their discretion.
 
+h4. TokenRevocationRequest
+
+* @(TF1)@ @TokenRevocationRequest@ is a type containing the token revocation request details sent to the "REST revokeTokens endpoint":/rest-api/#revokeTokens
+* @(TF2)@ @TokenRevocationRequest#key@ attribute is a string representing key to revoke (appId.Key).
```

Comment:
Why require the key to be passed in here? This is a method on an instantiated client lib instance, which will already have some auth params; if it has access to an api key it would presumably already have been instantiated with it?

(In the unusual event that someone needs to revoke things from multiple keys, they can always instantiate a new rest client lib for each one)

Reply:
@SimonWoolf The wording wasn't very descriptive and it wasn't necessary after all. I got appId.Key from here https://ably.com/docs/api/rest-api#revoke-tokens 
I suppose it is just the key associated with the current Rest client. \ I removed all key and references here 029fe8ab29162690d758649f60a587a4b4e62bc7